packages/ts/react-signals/src/FullStackSignal.ts

```diff
@@ -153,6 +189,26 @@ export abstract class FullStackSignal<T> extends Signal<T> {
       }
     });
 
-    paused = false;
+    this.#paused = false;
+  }
+
+  #connect() {
+    if (this.server.subscription !== undefined) {
```

Comment:
Why do we need this condition here?

Reply:
Yeah, good point. This probably cannot/should not happen, atm.
Later, when these connect/disconnect methods are going to be called every time the flux connection recovers (apart from the a first dependency subscribing to the signal), and in case of having an active subscription it should not create another one.

Reply:
I'll remove it for now.